packages/vaadin-avatar/test/avatar.test.js

```diff
@@ -15,16 +15,18 @@ describe('vaadin-avatar', () => {
   });
 
   describe('custom element definition', () => {
-    it('should be defined with correct tag name', () => {
-      expect(customElements.get('vaadin-avatar')).to.be.ok;
+    let tagName;
+
+    beforeEach(() => {
+      tagName = avatar.tagName.toLowerCase();
     });
 
-    it('should not expose class name globally', () => {
-      expect(window.AvatarElement).not.to.be.ok;
+    it('should be defined in custom element registry', () => {
+      expect(customElements.get(tagName)).to.be.ok;
```

Comment:
Same problem as for `vaadin-accordion`, this test won't be executed as `imgElement = avatar.shadowRoot.querySelector('img');` throws an `undefined` exception before.

Reply:
Updated the `beforeEach` accordingly.